### DISCUSSION TITLE
[Artix] Proper way to setup auto login on tty1

### DISCUSSION THREAD

**Discussion by omnigenous** [davmac314/dinit#335] [Q&A] [answered]
Currently I have to edit `/etc/dinit.d/tty1` file to enable auto login

`/etc/dinit.d/tty1`:
```
...
command = /usr/lib/dinit/agetty-default --autologin <USER> tty1
...
```

The only problem is after upgrading this files resets. Is there a other way to enable auto login that won't reset?

**Comment by konimex** [comment, marked as answer]
Since this is probably an Artix-specific issue (looking from the dir it points to), I'm going to step in.

You can copy `/etc/dinit.d/config/agetty-default` (if you are using Artix) to `/etc/dinit.d/config/agetty-tty1.conf`, after that, you can edit the newly-copied file and add `"--autologin <USER>"` to the `GETTY_ARGS=` var, it'll be automatically read at the next boot.

**Reply by omnigenous** [reply]
Thank you! Little typo, I think you meant `/etc/dinit.d/config/agetty-default.conf` instead of `/etc/dinit.d/config/agetty-default`
